discovery-provider/src/queries/get_users_cnode.py

```diff
@@ -0,0 +1,46 @@
+import logging # pylint: disable=C0302
+import sqlalchemy
+
+from src.utils.db_session import get_db_read_replica
+
+logger = logging.getLogger(__name__)
+
+def get_users_cnode(cnode_endpoint_string):
```

Comment:
can you throw in a comment for what this does

also i **think** the outer query is unnecessary, like you could do `select user_id, wallet, [1] as primary.... from () where "primary" = :primaryurl` etc but don't think it makes big diff

Reply:
most def, this needs some explanation